### DISCUSSION TITLE
PWMs for motifs in your tbl

### DISCUSSION THREAD

**Discussion by daxus4** [aertslab/SCENIC#292] [* General SCENIC questions] [answered]
Hi!

I want to use the pySCENIC pipeline with a custom cistarget_database. I used `create_cistarget_databases` to create my custom cistarget_databases. Everything works well.

I just have a problem. I want to do a motif enrichment using my ATAC peaks and the motifs contained in [this](https://resources.aertslab.org/cistarget/motif2tf/motifs-v9-nr.hgnc-m0.001-o0.0.tbl) tbl file, which is one of the your tbls.

Where can I find the PWM or similar for these motifs? Can you send me the link?

Thank you!

**Comment by s-aibar** [comment, marked as answer]
Dear @daxus4 ,

At the moment we do not provide the PWM for the motifs. However, if you are interested in one or a few motifs in particular, you can probably find their PWMs in the corresponding source (most motifs have the source database in their name, e.g. Homer, Jaspar, etc...)
